File: backend/src/main/java/com/taskDistributor/models/Team.java
package com.taskDistributor.models;

import javax.persistence.Entity;
import javax.persistence.Table;
import javax.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
@Table(name = "teams")
public class Team extends AbstractIdentifiable {

  @NotBlank
  private String teamName;

  private String description;
}

File: frontend/src/app/models/Team.ts
export class Team {
  id: number;
  teamName: string;
  description: string;

  constructor(
    id: number,
    teamName: string,
    description: string
  ) {
    this.id = id;
    this.teamName = teamName;
    this.description = description;
  }
}
